development/verify-locale-strings.js

```diff
@@ -161,11 +161,14 @@ async function verifyEnglishLocale (fix = false) {
   const englishLocale = await getLocale('en')
   const javascriptFiles = await findJavascriptFiles(path.resolve(__dirname, '..', 'ui'))
 
-  const regex = /'(\w+)'/g
+  const regex = /'(\w+)'|"(\w+)"/g
   const usedMessages = new Set()
   for await (const fileContents of getFileContents(javascriptFiles)) {
     for (const match of matchAll.call(fileContents, regex)) {
-      usedMessages.add(match[1])
+      const [, ...captureGroups] = match
+      for (const captureGroup of captureGroups) {
```

Comment:
That regex will match either the first or second capture group - not both. This looks like it would result in `null`/`undefined` being erroneously added to the `usedMessages` array.

Reply:
Yeah, I am constantly confusing myself with how regex captures iterate—I misread the docs for `matchAll`. I'll just go with your original suggestion.